src/Pulp/Files.purs

```diff
@@ -54,9 +60,23 @@ testGlobs :: Options -> AffN (Set String)
 testGlobs = globsFromOption "testPath"
 
 dependencyGlobs :: Options -> AffN (Set String)
-dependencyGlobs =
-  globsFromOption' (\path -> Path.concat [path, "purescript-*", "src"])
-                   "dependencyPath"
+dependencyGlobs opts = do
+  Project pro <- getOption' "_project" opts
+  -- If project file has a `set` property we assume it's a psc-package project file
```

Comment:
Would it make sense to extract this into a function `usingPscPackage :: Project -> Boolean` in `Pulp.Project`?